image/templates/helm/stackrox-central/templates/_central_setup.tpl

```diff
@@ -52,26 +52,35 @@
     Central's DB PVC config setup
   */}}
 {{ $dbVolumeCfg := dict }}
+{{ $dbBackupVolumeCfg := dict }}
 {{ if not $centralDBCfg.external }}
 {{ if $centralDBCfg.persistence.none }}
   {{ include "srox.warn" (list $ "You have selected no persistence backend. Every deletion of the StackRox Central DB pod will cause you to lose all your data. This is STRONGLY recommended against.") }}
   {{ $_ := set $dbVolumeCfg "emptyDir" dict }}
+  {{ $_ := set $dbBackupVolumeCfg "emptyDir" dict }}
 {{ end }}
 {{ if $centralDBCfg.persistence.hostPath }}
   {{ if not $centralDBCfg.nodeSelector }}
     {{ include "srox.warn" (list $ "You have selected host path persistence, but not specified a node selector. This is unlikely to work reliably.") }}
   {{ end }}
   {{ $_ := set $dbVolumeCfg "hostPath" (dict "path" $centralDBCfg.persistence.hostPath) }}
+  {{ $_ := set $dbBackupVolumeCfg "hostPath" (printf "%s/backup" (dict "path" $centralDBCfg.persistence.hostPath)) }}
 {{ end }}
 {{/* Configure PVC if either any of the settings in `centralDB.persistence.persistentVolumeClaim` are provided,
      or no other persistence backend has been configured yet. */}}
 {{ if or (not (deepEqual $._rox._configShape.central.db.persistence.persistentVolumeClaim $centralDBCfg.persistence.persistentVolumeClaim)) (not $dbVolumeCfg) }}
   {{ $dbPVCCfg := $centralDBCfg.persistence.persistentVolumeClaim }}
+  {{ $dbBackupPVCCfg := $centralDBCfg.persistence.persistentVolumeClaim }}
   {{ $_ := include "srox.mergeInto" (list $dbPVCCfg $._rox._defaults.dbPVCDefaults (dict "createClaim" (or .Release.IsInstall (eq $._rox._renderMode "centralDBOnly")))) }}
+  {{ $_ := include "srox.mergeInto" (list $dbBackupPVCCfg $._rox._defaults.dbPVCDefaults (dict "createClaim" (or .Release.IsInstall (eq $._rox._renderMode "centralDBOnly") $centralDBCfg.persistence.createBackup ))) }}
```

Comment:
The string `createBackup` does not exist anywhere else (apart from some apparently unrelated UI code). What am I missing?

Reply:
The idea is to have it as a knob for Helm-only installations. That way someone could create a backup volume manually, then set `createBackup` when doing `helm upgrade`.

Reply:
I'm going to separate Helm-only bits from this PR, turning it into Operator-only changes. Then I'll follow up with another PR to support Helm-only backup volume. How does it sound?